cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -199,7 +195,8 @@ bool AudioEngineImpl::init()
         // get the engine interface, which is needed in order to create other objects
         result = (*_engineObject)->GetInterface(_engineObject, SL_IID_ENGINE, &_engineEngine);
         if(SL_RESULT_SUCCESS != result){ ERRORLOG("get the engine interface fail"); break; }
-
+        
+    #if CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID
```

Comment:
No need for _engineObject at all on the openharmony platform?

Reply:
It's needed by AudioDecoderSLES to decode audio files in some formats.
![image](https://github.com/cocos/engine-native/assets/41882593/256eaf91-9932-4b50-a938-1f2639f1ded3)